emission/net/api/cfc_webapp.py

```diff
@@ -383,17 +384,14 @@ def getCarbonCompare():
   return choice.getResult(user_uuid)
 
 
-#Pulling public data from the server  
-@get('/getData')
-def getData():
+# Pulling public data from the server  
+@get('/eval/publicData/timeseries')
+def getPublicData():
   from_date = request.query.from_date
   to_date = request.query.to_date
+  from_ts = arrow.get(from_date).timestamp
```

Comment:
Instead of passing in the date to `getPublicData` and YYYY-MM-DD, why not just pass it directly as a UTC timestamp. Unambiguous. Clear.